plugins/sfEadPlugin/modules/sfEadPlugin/templates/indexSuccessBody.xml.php

```diff
@@ -81,23 +81,18 @@
 
 <archdesc <?php echo $ead->renderLOD($resource, $eadLevels); ?> relatedencoding="<?php echo $ead->getMetadataParameter('relatedencoding'); ?>">
   <?php
+    $resourceVar = 'resource';
```

Comment:
Is this to avoid HTML Purifier from messing things up?

Reply:
@mcantelon are you asking if the `$resourceVar` variable is to avoid HTML Purifier messing things up?  If so, I'm not really sure why it uses a variable variable pattern here, but it predates HTML Purifier.   It looks like it was introduced by _you_ 10 years ago: https://github.com/artefactual/atom/blame/0971f2634c3371418fa28f1e2e2c8a9292477758/plugins/sfEadPlugin/modules/sfEadPlugin/templates/indexSuccessBody.xml.php 🤔

Reply:
Lolol..., I was probably trying to avoid some kind of escaping-ish issue then. ;)